articles/communication-services/quickstarts/rooms/includes/rooms-quickstart-javascript.md

```diff
@@ -72,11 +71,11 @@ Create a new `room` with default properties using the code snippet below:
 
 ```javascript
 // create a room with the request payload
-const createRoom = await roomsClient.createRoom(createRoomRequest);
+const createRoom = await roomsClient.createRoom(createRoomOptions);
```

Comment:
Do need to add createRoomOptions obj as well?
Copied from the repo 
// options payload to create a room
  const createRoomOptions = {
    validFrom: validFrom,
    validUntil: validUntil,
    participants: [
      {
        id: user1.user,
        role: "Attendee",
      },
    ],
  };